cypress/support/commands.js

```diff
@@ -20,7 +20,7 @@ const TEST_PRIVATE_KEY = '0x60aec29d4b415dfeff21e7f7d07ff2aca0e26f129fe52fc4e86f
 // address of the above key
 export const TEST_ADDRESS_NEVER_USE = new Wallet(TEST_PRIVATE_KEY).address
 
-export const TEST_ADDRESS_NEVER_USE_SHORTENED = `0x...${TEST_ADDRESS_NEVER_USE.substr(-4, 4)}`
+export const TEST_ADDRESS_NEVER_USE_SHORTENED = `0x...${TEST_ADDRESS_NEVER_USE.substring(-4, 4)}`
```

Comment:
<img width="232" alt="Screen Shot 2022-05-09 at 09 21 20" src="https://user-images.githubusercontent.com/97418926/167408738-31db4716-ae28-40bb-adc1-251851796d6f.png">

Could you doublecheck this update? It seems 2 methods don't use the same api.

Reply:
Done